browser/net/brave_network_audit_browsertest.cc

```diff
@@ -320,6 +334,7 @@ IN_PROC_BROWSER_TEST_F(BraveNetworkAuditTest, BasicTests) {
       WaitForTimeout(kMaxTimeoutPerLoadedURL);
     }
   }
+  // TODO(sko) What should we do for this? is this comment still valid?
   // Currently, builtin panel item is one.
   // If it's increased, --test-launcher-time should be increased also.
```

Comment:
@simonhong I ran into this line but not sure what this means. Could you give me some advices?

Reply:
the argument in `//brave/package.json` is big enough `--test-launcher-timeout=1900000`